src/markdown_generators.js

```diff
@@ -551,53 +549,100 @@ const generatePdfMarkdown = (file, courseData) => {
   return `---\n${yaml.safeDump(pdfFrontMatter)}---\n`
 }
 
-const generateCourseFeaturesMarkdown = (page, courseData) => {
+const generateVideoGalleryMarkdown = (page, courseData) => {
   let courseFeaturesMarkdown = ""
-  if (page.hasOwnProperty("is_image_gallery")) {
-    if (page["is_image_gallery"]) {
-      const images = courseData["course_files"].filter(
-        file =>
-          file["parent_uid"] === page["uid"] && file["type"] === "OCWImage"
+  const videos = Object.values(courseData["course_embedded_media"]).filter(
+    obj => obj["parent_uid"] === page["uid"]
+  )
+  if (videos.length > 0) {
+    const videoDivs = []
+    videos.forEach(video => {
+      const videoArgs = {
+        href: helpers.pathToChildRecursive(
+          `/courses/${courseData.short_url}/sections`,
+          video,
+          courseData
+        ),
+        section: helpers.htmlSafeText(
```

Comment:
This is a bit of a nitpick and shouldn't hold up the merging of this, but maybe we should call this field "subtitle" or something more generic?  Right now this only displays the section, but thinking about it as a generic "video gallery item" shortcode it might be better to be more generic here.